apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -57,15 +61,18 @@ const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsu
                 if (!organization) {
                     throw new Error(`${NOT_FOUND_ERROR}organization] Organization not found for this id`)
                 }
-                //TODO(zuch): Ask about wrong logic - resident unit name do not match billing account unitName
-                const unitName = get(resident, ['unitName'])
+
+                const unitName = get(resident, 'unitName', null)
+
+                const paymentCategory = get(extra, 'paymentCategory', null)
```

Comment:
why null? category is not required? then it will search for all consumers without category....is it correct case?

Reply:
Category is not required. We do not use paymentCategories in business logic yet, and hopefully don't plan